### DISCUSSION TITLE
Spreading awareness of PEP 752

### DISCUSSION THREAD

**Discussion by ofek** [pytest-dev/pytest#12732] [Ideas]
Hello team! I just wanted to bring to your attention a new PEP about package repository namespaces where I used your packages/community as a motivating example 🙂

We would love your feedback in the current discussion: https://discuss.python.org/t/pep-752-package-repository-namespaces/61227

**Comment by RonnyPfannschmidt** [comment]
Thanks for the note, this looks pretty interesting, I hope to get to a deep read this weekend as the finer details are important 

From my initial skimming I'm already impressed and like the ideas and Nuance 

I'll still try to see if a devil is hiding in the details

**Reply by ofek** [reply]
FYI the proposal has been split into a pure specification one (752) and another for policy/operational stuff. It would be very helpful if you could leave your feedback! I'm trying to get explicit communication that projects and communities would benefit from this.

New threads:

* [PEP 752](https://discuss.python.org/t/pep-752-implicit-namespaces-for-package-repositories/63192)
* [PEP 755](https://discuss.python.org/t/pep-755-implicit-namespace-policy-for-pypi/63191)
